File: mcconaha-steve/calculator-coin-combo.js
function coinCombo (cents) {
	//define cent variables
  let pennies = 0;
  let nickels = 0;
  let dimes = 0;
  let quarters = 0;
  let orgCent = cents;
  //quarter = cents / 25 rounded down
  quarters = Math.floor(cents/25);
  //redefine cents with each pass through denomination
  //Deduct the denomination * value from the original cents
  cents = cents - (quarters*25);
  console.log(`Of ${orgCent} cents I have ${quarters} quarters`);
  //quarter = cents / 10 rounded down
  //Deduct the denomination * value from the original cents
  dimes = Math.floor(cents/10);
  cents = cents - (dimes*10);
  console.log(`Of ${orgCent} cents I have ${dimes} dimes`);
  //quarter = cents / 5 rounded down
  //Deduct the denomination * value from the original cents
  nickels = Math.floor(cents/5);
  cents = cents - (nickels*5);
  console.log(`Of ${orgCent} cents I have ${nickels} nickels`);
  //quarter = cents / 1 rounded down
  //Deduct the denomination * value from the original cents
  pennies = Math.floor(cents/1);
  cents = cents - (pennies*1);
  console.log(`Of ${orgCent} cents I have ${pennies} pennies`);

  return [pennies, nickels, dimes, quarters];
}

File: andrews-zack/calculator-coin-combo.py
import math

def coin_combo(c):
    q = 0
    d = 0
    n = 0
    p = 0
    if c >= 25:
        q = math.floor(c/25)
        c = c - (q*25)
    if c >= 10:
        d = math.floor(c/10)
        c = c - (d*10)
    if c >= 5:
        n = math.floor(c/5)
        c = c - (n*5)
    if c >= 1:
        p = c
    return [p, n, d, q]
